pkgs/by-name/pi/picom-pijulius/package.nix

```diff
@@ -2,17 +2,18 @@
   picom,
   lib,
   fetchFromGitHub,
-  pcre
+  pcre,
+  nix-update-script
 }:
 picom.overrideAttrs (previousAttrs: {
   pname = "picom-pijulius";
-  version = "8.2-unstable-2024-04-30";
+  version = "Next-unstable-2024-06-06";
 
   src = fetchFromGitHub {
     owner = "pijulius";
     repo = "picom";
-    rev = "e7b14886ae644aaa657383f7c4f44be7797fd5f6";
-    hash = "sha256-YQVp5HicO+jbvCYSY+hjDTnXCU6aS3aCvbux6NFcJ/Y=";
+    rev = "083421980af3983a4ca8bd56373303dc0282b395";
```

Comment:
Is there a reason we are not using the latest commit here?

Reply:
That is the latest commit, at least according to nix-update